atkinson/errors/reporters/__init__.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env python
+"""
+General error report classes
+"""
+
+from abc import ABC, abstractmethod
+
+
+class BaseReport(ABC):
+    """ Abstract base class for error reports """
+
+    @classmethod
+    @abstractmethod
+    def new(cls, title, description, config):
+        """ Create a new report and return a report object """
+
+    @classmethod
+    @abstractmethod
+    def get(cls, report_id, config):
+        """ Get a current report object by ID """
+
+    @property
+    @abstractmethod
+    def report_id(self):
```

Comment:
Unless it is a reserved word, perhaps just call this 'id'?

Reply:
'id' is a reserved word in python https://docs.python.org/3/library/functions.html#id

Reply:
Well, maybe just it leave then.  I just did a quick test, and adding an id property seems to work fine, since it is never referenced as a method call.  That said, it may be confusing to use that as a property when there is also a function with the same name.